File: net_to_java/output/BookStore/BookStoreApp/Models/Order.cs
// Assisted by watsonx Code Assistant 
// watsonx Code Assistant did not check whether this code suggestion might be similar to third party code.
using System;
using System.Collections.Generic;

namespace BookStoreApp.Models
{
    public class Order
    {
        public Guid Id { get; set; }
        public Guid CustomerId { get; set; }
        public DateTime OrderDate { get; set; }
        public OrderStatus Status { get; set; }
        public decimal TotalAmount { get; set; }
        public Address ShippingAddress { get; set; }
        public PaymentStatus PaymentStatus { get; set; }
        public List<OrderItem> OrderItems { get; set; }
    }
}

File: net_to_java/output/BookStore/ClientApp/BookStoreApp/src/types/order.ts
// Assisted by watsonx Code Assistant 
// watsonx Code Assistant did not check whether this code suggestion might be similar to third party code.
export interface Order {
  id: string;
  customerId: string;
  orderDate: Date;
  status: OrderStatus;
  totalAmount: number;
  shippingAddress: Address;
  paymentStatus: PaymentStatus;
  orderItems: OrderItem[];
}

export enum OrderStatus {
  Created = 'Created',
  Processing = 'Processing',
  Shipped = 'Shipped',
  Delivered = 'Delivered',
  Cancelled = 'Cancelled',
}

export enum PaymentStatus {
  Pending = 'Pending',
  Failed = 'Failed',
  Succeeded = 'Succeeded',
}

export interface OrderItem {
  bookId: string;
  quantity: number;
}